src/namespaces/cdc/cdc.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Expr } from "../..";
+import { CypherProcedure } from "../../procedures/CypherProcedure";
+import { normalizeExpr, normalizeList } from "../../utils/normalize-variable";
+
+/** Acquire a change identifier for the last committed transaction
+ * @see [Neo4j Documentation](https://neo4j.com/docs/cdc/current/)
```

Comment:
```suggestion
 * @see [Neo4j Documentation](https://neo4j.com/docs/cdc/current/procedures/current/)
```

Reply:
I wonder if it's worth mentioning in the TSDoc that this procedures are available only from Neo4j Enterprise Edition 5.13